posts/2020/03/documentation-as-a-way-to-build-community.md

```diff
@@ -0,0 +1,60 @@
+
+Even when the community is welcoming, documentation is often seen as a "good first issue", meaning that the docs end up being written by the least experienced contributors in the community. This can have its advantages, as it may give voice to the users who are experiencing difficulties, improving the communication between the projects and its community. However, it may transfer the responsibility of one of the most crucial aspects of any project to novice users, who have neither the knowledge or the experience to make decisions about it.
+
+## What can we do about it?
+
+So if expert users and developers are too busy to write docs, or just don't want to, how can we address this problem? 
+
+As much as the culture is changing, and many people are talking about the importance of documentation, we still have [technical debt](https://en.wikipedia.org/wiki/Technical_debt) related to projects that have been around for a while. This is not something we can overlook: having good documentation is the difference between being successful or not in solving our users' (and clients') problems. Furthermore, it is also crucial if we aim to improve reproducibility and transparency issues in science and data analytics.
+
+In this sense, grants like the one NumPy just received are extremely important to create momentum around documentation and reshape the community into one that values those contributions. 
+
+### Short and long-term goals
+
+As part of our work related to the CZI grant, we have submitted a [NumPy Enhancement Proposal (NEP)](https://numpy.org/neps/nep-0044-restructuring-numpy-docs.html) that proposes a restructure the NumPy documentation, with the goal of making it more organized and discoverable for beginners and experienced users. 
+
+In practical terms, we propose reorganizing the docs into the four categories mentioned in Daniele Procida's article ["What nobody tells you about documentation"](https://www.divio.com/blog/documentation), namely Tutorials, How-Tos, Reference Guide and Explanations. We believe that this will have several consequences:
+- Improving on the quality and discoverability of the documentation as a whole;
+- Showing a clearer difference between documentation aimed at different users (novices vs. experts, for example)
+- Giving users more opportunities to contribute, generating content that can be shared directly on NumPy's official documentation
+- Diversifying our contributor's base, allowing people from different levels of expertise and different life experiences to contribute. This is also extremely important so that we have a better understanding of our community and are able to be accessible, unbiased and welcoming to all people.
+
```

Comment:
There's something you can add here that's in the grant related to the docs-core-team issue you identified earlier: creating a "documentation team" as a first-class team in the project. It's governance deliverable 1 in the proposal: _" Define explicit roles in the project structure and adding a mechanism for devolved decision making on code, documentation, website, and fundraising."_

Reply:
Thanks!